src/main/resources/messages.properties

```diff
@@ -0,0 +1,19 @@
+CREATED=Employee Created
+NO_DATA_FOUND=No Records Found
```

Comment:
Message should have more information. For example if a getEmployee has been called with a employee id which doesn't exit, then user should get message as "Employee with id #1 not found in out record."